src/base.js

```diff
@@ -342,6 +342,14 @@ VRDisplay.prototype.fireVRDisplayPresentChange_ = function() {
   window.dispatchEvent(event);
 };
 
+VRDisplay.prototype.fireVRDisplayConnect_ = function() {
+  // Important: unfortunately we cannot have full spec compliance here.
+  // CustomEvent custom fields all go under e.detail (so the VRDisplay ends up
+  // being e.detail.display, instead of e.display as per WebVR spec).
+  var event = new CustomEvent('vrdisplayconnect', {detail: {display: this}});
```

Comment:
FWIW, I was going off of the spec for `vrdisplayconnect` -- and good call on IE, I'll add a follow up issue for that, agreed low priority due to IE11, filed #249